native/cocos/platform/openharmony/OpenHarmonyPlatform.h

```diff
@@ -73,18 +73,22 @@ class OpenHarmonyPlatform : public UniversalPlatform {
     void onSurfaceDestroyed(OH_NativeXComponent* component, void* window);
     void onSurfaceHide();
     void onSurfaceShow(void* window);
+    void dispatchMouseWheelCB(std::string eventType, float localX, float localY, float offsetY);
 
     static void onMessageCallback(const uv_async_t* req);
     static void timerCb(uv_timer_t* handle);
 
     OH_NativeXComponent* _component{nullptr};
     OH_NativeXComponent_Callback _callback;
+    OH_NativeXComponent_MouseEvent_Callback _mouseCallback;
```

Comment:
It's better to intialize callback to `nullptr`.
```suggestion
    OH_NativeXComponent_MouseEvent_Callback _mouseCallback{nullptr};
```
```c++
OH_NativeXComponent_Callback _callback{nullptr};
```

Reply:
fixed